html-css/week1/Ali Sharifi/index.html

```diff
@@ -0,0 +1,94 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+</head>
+<body class="container">
+    <!-- NavBar -->
+    <nav class="navbar">
+        <ul>
+            <li><a href="#header">Home</a></li>
+            <li><a href="#about">About</a></li>
+            <li><a href="#skills">Skills</a></li>
+            <li><a href="#portfolio">Portfolio</a></li>
+            <li><a href="#contact">Contact</a></li>
+        </ul>
+    </nav>
+
+    <!-- header -->
+    <header id="header">
+        <div class="background">
+            <img  class="avatar" src="avatar.jpeg" alt="">
+            <h1 class="t-word"><span class="s-word">Front-end Developer</span><br />Ali sharifi</h1>
+          </div>
+    </header>
+
+    <!-- About-->
+    <section id="about">
```

Comment:
It would be prefered to create a class with the same name, and move the CSS from the ID to the class.
```suggestion
    <section id="about" class="about">
```